content/news/2021-XX-XX-fsf-statement.md

```diff
@@ -0,0 +1,19 @@
+title: "A Statement on Richard Stallman"
+authors: Owen Williams
```

Comment:
Maybe we should sign this as "The Mixxx Team"?

Reply:
yes definitely